### PR TITLE
BP reference: document `rhsm` and `rpm` customizations (COMPOSER-2308)

### PR DESCRIPTION
**Pull request by thozza**

### CODE REVIEW

File: docs/user-guide/01-blueprint-reference.md [hunk 1]
```diff
@@ -357,6 +357,74 @@ append = "nosmt=force"
 </TabItem>
 </Tabs>
 
+### Subscription-manager (RHSM) 🔵 {#rhsm}
+
+An optional set of configuration options for the Red Hat Subscription Manager (RHSM) client.
+
+The customization expects that `subscription-manager` package is installed in the image.
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: For newbies maybe add a link to the package section? (no merge blocker for me though)

File: docs/user-guide/01-blueprint-reference.md [hunk 2]
```diff
@@ -357,6 +357,74 @@ append = "nosmt=force"
...
+
+The customization is not supported on Fedora distribution images.
+
+To configure RHSM in the final image:
+
+<Tabs values={tabValues} >
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: please use this, this visually looks nicer here:
```suggestion
<Tabs values={tabValuesOnPremiseOnly} >
```

File: docs/user-guide/01-blueprint-reference.md [hunk 3]
```diff
@@ -357,6 +357,74 @@ append = "nosmt=force"
...
+
+### RPM configuration 🔵 {#rpm}
+
+An optional set of RPM configuration options. Currently, only importing RPM GPG keys from a file in the image is supported.
+
+`files` is a list of strings containing the paths to the GPG keys to import. The files must be present in the image.
+
+To configure RPM in the final image:
+
+<Tabs values={tabValues} >
```
Comment (schuellerf) [inline docs/user-guide/01-blueprint-reference.md]: also here
```suggestion
<Tabs values={tabValuesOnPremiseOnly} >
```

**Review by schuellerf — CHANGES_REQUESTED** [review]

File: docs/user-guide/01-blueprint-reference.md (hunk 1 above)
Comment (thozza) [reply docs/user-guide/01-blueprint-reference.md]: Thanks, I added the link and reworded it a bit.

**Review by schuellerf — APPROVED** [review]
LGTM, thanks for the updates

**Pull request merged**